src/jobs/release-package.yml

```diff
@@ -79,6 +79,9 @@ steps:
   - run:
       name: Install dependencies
       command: yarn
+  - run:
```

Comment:
Do you think there'd be a scenario where we don't want this step to run? If so we might want to put a conditional on it

Reply:
I don't think so. [`yarn install` is already supposed to do this](https://classic.yarnpkg.com/en/docs/cli/dedupe/) but clearly doesn't. If a certain dependency needs a specific version of something they should just pin to it. This will just affect version ranges.

Reply:
More info here: https://github.com/scinos/yarn-deduplicate#duplicated-packages